### PR TITLE
Change tmp dir to HOME path

### PR DESCRIPTION
**Pull request by yotamloe**

### CODE REVIEW

**Review by yyyogev — APPROVED** [review]
Great work!

File: store/config.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ const (
 	customListenerParam = "CUSTOM_LISTENER_URL"
 	customAPIParam      = "CUSTOM_API"
 	usRegionCode        = "us"
+	customQueueDirParam  = "CUSTOM_QUEUE_PATH"
```
Comment (Doron-Bargo) [inline store/config.go:23]: fix liniting

File: store/config.go [hunk 2]
```diff
@@ -29,6 +30,8 @@ type LogzioConfig struct {
 	APIToken          string `yaml:"apiToken"`
 	CustomListenerURL string `yaml:"customListenerUrl"`
 	CustomAPIURL      string `yaml:"customAPIUrl"`
+	CustomQueueDir     string `yaml:"customQueueDir"`
```
Comment (Doron-Bargo) [inline store/config.go:33]: fix linting

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Please
1. Update readme with the new parameter
2. Add test as part of the initialization of queue path that the directory is accessible

File: store/config.go (hunk 2 above)
Comment (yotamloe) [reply store/config.go]: @Doron-Bargo Please check out new [commit](https://github.com/logzio/jaeger-logzio/pull/23/commits/a23fb942b7dfd1e2355187932809a165b18a40a1)

File: store/config.go (hunk 1 above)
Comment (yotamloe) [reply store/config.go]: @Doron-Bargo Please check out new [commit](https://github.com/logzio/jaeger-logzio/pull/23/commits/a23fb942b7dfd1e2355187932809a165b18a40a1)

**Review by Doron-Bargo — APPROVED** [review]
Amazing work

**Pull request merged**